liquid.py

```diff
@@ -327,16 +396,94 @@ def get_job(hcl_path, substitutions={}):
     return template.safe_substitute(set_volumes_paths(substitutions))
 
 
+def gc():
+    """Stop collections jobs that are no longer declared in the ini file."""
+
+    nomad_jobs = nomad.jobs()
+
+    for job in nomad_jobs:
+        if job['ID'].startswith('collection-'):
+            collection_name = job['ID'][len('collection-'):]
+            if collection_name not in config.collections and job['Status'] == 'running':
+                log.info('Stopping %s...', job['ID'])
+                nomad.stop(job['ID'])
+
+
+def get_collections_to_purge():
+    """Returns a set of collections to purge."""
+
+    ini_collections = set(config.collections.keys())
+
+    collections_dir = Path(config.liquid_volumes) / 'collections'
+    volumes = set([subdir.name for subdir in collections_dir.iterdir() if subdir.is_dir()])
+
+    return volumes - ini_collections
+
+
+def purge_collection(name):
+    """Purge data for the given collection, remove it from hoover search."""
+
+    try:
```

Comment:
This passthrough makes no sense here. Please remove the `try`.

Reply:
It makes sense because the exception stacktrace does not contain the output from the command run in the hoover-search service. I just forgot to raise again the exception, which I'll do.